src/components/Nav.svelte

```diff
@@ -57,11 +57,17 @@ $: totalNumUnreads = $unreads.reduce((sum, { count }) => sum + count, 0)
         {/if}
 
         <li class="nav-item">
-          <a href="/#/requests" class="nav-link" class:active={$location === '/requests'}>
+          <a href="/#/requests" class="nav-link" class:active={$location.startsWith('/requests')}>
```

Comment:
Nice catch!

Reply:
Thanks, I should've done it on `develop` though, sorry about that.